cmd/influxd/config.go

```diff
@@ -34,11 +34,12 @@ const (
 
 // Config represents the configuration format for the influxd binary.
 type Config struct {
-	Hostname          string `toml:"hostname"`
-	BindAddress       string `toml:"bind-address"`
-	ReportingDisabled bool   `toml:"reporting-disabled"`
-	Version           string `toml:"-"`
-	InfluxDBVersion   string `toml:"-"`
+	Hostname              string `toml:"hostname"`
+	BindAddress           string `toml:"bind-address"`
+	ReportingDisabled     bool   `toml:"reporting-disabled"`
+	AuthenticationEnabled bool   `toml:"authentication-enabled"`
```

Comment:
I think this would look better (and be more extensible in the future) if it was:

```
Authentication struct {
    Enabled bool `toml:"enabled"`
} `toml:"authentication"`
```

I could see having other authentication types in the future and they could all go under this section.